EarthBnB/pom.xml

```diff
@@ -52,15 +62,10 @@
 			<version>2.3.3.RELEASE</version>
 		</dependency>
 		<dependency>
-			<groupId>org.springframework.data</groupId>
-			<artifactId>spring-data-jpa</artifactId>
-			<version>2.3.2.RELEASE</version>
```

Comment:
We had version 2.3.3 and version 2.3.2 of spring-data-jpa... so I removed the old one. Everything else changed not mentioned in my comments appears to be formatting or moved to make room for additions and removals of items.